fuzz/fuzz_targets/compile.rs

```diff
@@ -1,14 +1,20 @@
 #![no_main]
```

Comment:
This file became accidentally executable I believe

Reply:
I have no idea why my machine keeps doing this T.T